docs/agent/changes.md

```diff
@@ -98,27 +95,80 @@ The agent has a new set of command-line options:
 | stopservice     | stops the agent within the service control manager |
 | version         | Print the version info |
 
-#### Gui
-* The browser session must be restarted each time the Agent service is restarted.  This will be fixed in an upcoming beta.
+To run a sub-command, the Agent binary must be invoked like this:
+```
+<path_to_agent_bin> <sub_command> <options>
+```
+
+Some options have their own set of flags and options detailed in a help message.
+For example, to see how to use the `check` sub-command, run:
+```
+<agent_binary> check --help
+```
+
+### Linux
+
+There are a few major changes:
+
+* only the _lifecycle commands_ (i.e. `start`/`stop`/`restart`/`status` on the Agent service) should be run with `sudo service`/`sudo initctl`/`sudo systemctl`
+* all the other commands need to be run with the `datadog-agent` command, located in the `PATH` (`/usr/bin`) by default. The binary `dd-agent` is not available anymore.
+* the `info` command has been renamed `status`
+* the Agent 6 does not ship a SysV-init script (previously located at `/etc/init.d/datadog-agent`)
+* The configuration GUI is disabled by default on Linux-based systems.
```

Comment:
the GUI on Linux isn't really a change (there never was a GUI on Linux for A5). Could we move this note to the `GUI` section?